client/pages/ForgotPassword/index.js

```diff
@@ -0,0 +1,103 @@
+import React, { useContext, useState } from 'react';
+import { Formik } from 'formik';
+import axios from 'axios';
+import InputForm from '@components/InputForm';
+import Button from '@components/Button';
+import { ForgotPasswordValidator } from '@clientValidators/Auth';
+
+const ForgotPassword = () => {
+    const [errorState, setErrorState] = useState(false);
```

Comment:
There's absolutely no need for `errorState` and `errorMessage` or `successMessage` and `successState`

Formik already has inbuilt support for displaying error messages so no need to have them in this component's state. As for the success, remove it for now. We'll have a global flash context which will display a toast.